{
  "id": "MAL-2024-5284",
  "modified": "2024-10-24T01:01:58Z",
  "published": "2024-06-25T13:36:36Z",
  "aliases": [
    "SNYK-PYTHON-JUDYBADVANCED-5292143"
  ],
  "summary": "Malicious code in judyb-advanced (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "judyb-advanced",
        "purl": "pkg:pypi/judyb-advanced"
      },
      "versions": [
        "0.1.0"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/judyb-advanced/MAL-2024-5284.json"
      }
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://security.snyk.io/vuln/SNYK-PYTHON-JUDYBADVANCED-5292143"
    },
    {
      "type": "ARTICLE",
      "url": "https://www.fortinet.com/blog/threat-research/supply-chain-attack-via-new-malicious-python-packages"
    }
  ]
}